domain-management/src/main/java/org/jboss/as/domain/management/access/AccessIdentityResourceDefinition.java

```diff
@@ -170,7 +172,7 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
 
         @Override
         protected boolean requiresRuntime(OperationContext context) {
-            return true;
+            return context.getProcessType() != ProcessType.EMBEDDED_SERVER || context.getRunningMode() != RunningMode.ADMIN_ONLY;
```

Comment:
This change matches the check for the management interfaces, I have tried to match the interface as it is important if they come up this comes up.